apps/mobile/src/app/wallet/settings/index.tsx

```diff
@@ -18,20 +32,81 @@ export default function SettingsScreen() {
     <Box flex={1} backgroundColor="ink.background-primary">
       <ScrollView
         contentContainerStyle={{
-          paddingHorizontal: theme.spacing['3'],
+          paddingHorizontal: theme.spacing['5'],
           paddingTop: theme.spacing['5'],
           paddingBottom: theme.spacing['5'] + bottom,
           gap: theme.spacing[5],
         }}
       >
-        <Text>{t`Hello`}</Text>
-        <Button
+        <Cell
+          title={t`Wallets and accounts`}
+          subtitle={t`Add, configure and remove`}
+          Icon={WalletIcon}
           onPress={() => {
             router.navigate(APP_ROUTES.WalletWalletsSettings);
           }}
-          buttonState="default"
-          title={t`Wallets (temp button)`}
         />
+        <Divider />
+        <Cell
+          title={t`Display`}
+          subtitle={t`Theme, bitcoin unit and more`}
+          Icon={SunIcon}
+          onPress={() => {
+            router.navigate(APP_ROUTES.WalletSettingsDisplay);
+          }}
+        />
+        <Cell
+          title={t`Security`}
+          subtitle={t`Analytics and app authentication`}
+          Icon={ShieldIcon}
+          onPress={() => {
+            router.navigate(APP_ROUTES.WalletSettingsSecurity);
+          }}
+        />
+        <Cell
+          title={t`Networks`}
+          subtitle={t`Mainnet, testnet or signet`}
+          Icon={GlobeTiltedIcon}
+          onPress={() => {
+            router.navigate(APP_ROUTES.WalletSettingsNetworks);
+          }}
+        />
+        <Cell
+          title={t`Notifications`}
+          subtitle={t`Push and email notifications`}
+          Icon={BellIcon}
+          onPress={() => {
+            router.navigate(APP_ROUTES.WalletSettingsNotifications);
+          }}
+        />
+        <Cell
+          title={t`Help`}
+          subtitle={t`Support, guides and articles`}
+          Icon={SupportIcon}
+          onPress={() => {
+            router.navigate(APP_ROUTES.WalletSettingsHelp);
+          }}
+        />
+        <Accordian
+          label={t`More options`}
+          content={
+            <>
+              <Cell title={t`Contacts`} Icon={SupportIcon} />
+              <Cell title={t`Fees`} Icon={SettingsSliderThreeIcon} />
+            </>
+          }
+        />
+        {/* TODO: Replace with shared Footer */}
```

Comment:
I'll add the footer and can do this in https://github.com/leather-io/issues/issues/228 in case you want to add that to the comment